Tests/MongoSwiftTests/CodecTests.swift

```diff
@@ -21,7 +21,7 @@ final class CodecTests: XCTestCase {
     }
 
     func testEncodeListDatabasesOptions() throws {
-        let options = ListDatabasesOptions(filter: Document(["a": 10]), nameOnly: true, session: ClientSession())
+        let options = ListDatabasesOptions(filter: ["a": 10], nameOnly: true, session: ClientSession())
```

Comment:
cool, you don't even need an `as`!

Reply:
yup, since the `filter` param is always a `Document` the compiler infers it 😀